core/services/log/registrations.go

```diff
@@ -201,6 +232,15 @@ func (r *registrations) sendLog(log types.Log, orm ORM, latestHead *models.Head,
 			continue
 		}
 
+		if len(metadata.filters) > 0 && len(log.Topics) > 1 {
+			logger.Tracew("LogBroadcaster: Checking filters")
```

Comment:
As in your previous PR I would definitely suggest not logging every log that comes in (could explode in production or make debugging very difficult)

Reply:
oh yes i added these here before i removed traces from the other PR - thanks.